dashboard-improvements/README.md

```diff
@@ -0,0 +1,151 @@
+# Improving dashboard
+
+## Main points
+
+- Informing users about results of SRPM, COPR and Koji builds or Testing Farm
+- Replace service "Build Result" webpage with dashboard
+  - Deprecation of webpage in service
+- Database connection for dashboard
+  - Consider sensitivity of information provided (even with read-only access)
```

Comment:
AFAIK only allowlist is sensitive to some extent b/c it contains real logins and namespaces, the rest should be public information